openedx/core/djangoapps/course_live/plugins.py

```diff
@@ -0,0 +1,66 @@
+"""
+Course app configuration for live.
+"""
+from typing import Dict, Optional
+
+from django.contrib.auth import get_user_model
+from django.utils.translation import gettext_noop as _
+from opaque_keys.edx.keys import CourseKey
+
+from openedx.core.djangoapps.course_apps.plugins import CourseApp
+from openedx.core.djangoapps.course_live.config.waffle import ENABLE_COURSE_LIVE
+from .models import CourseLiveConfiguration
+
+User = get_user_model()
+
+
+class LiveCourseApp(CourseApp):
+    """
+    Course App config for Live.
+    """
+
+    app_id = "live"
+    name = _("Live")
+    description = _("Enable in-platform video conferencing by configuring live")
+    documentation_links = {
+        # TODO: add the actual documentation link once it exists
+        "learn_more_configuration": '',
```

Comment:
This Docs can be used here
https://support.zoom.us/hc/en-us/sections/4422131421581

Reply:
this link needs to be to the edx documentation